packages/broker/src/ConfigWizard.ts

```diff
@@ -260,11 +262,11 @@ export const createStorageFile = async (config: any, answers: inquirer.Answers):
 
 export const startBrokerConfigWizard = async(): Promise<void> => {
     try {
-        const answers = await inquirer.prompt(prompts)
-        const config = getConfigFromAnswers(answers)
-        if (answers.revealGeneratedPrivateKey) {
-            logger.info(`This is your node\'s private key: ${config.ethereumPrivateKey}`)
-        }
+        let config = { ... CONFIG_TEMPLATE, plugins: { ... CONFIG_TEMPLATE.plugins } }
+        const ethereumAnswers = await inquirer.prompt(privateKeyPrompts)
+        config = getEthereumConfigFromAnswers(ethereumAnswers, config)
+        const pluginAnswers = await inquirer.prompt([pluginSelectorPrompt, ...pluginPrompts])
+        config = getPluginsConfigFromAnswers(pluginAnswers, config)
         const nodeAddress = new Wallet(config.ethereumPrivateKey).address
```

Comment:
While we are changing the order of messaging, maybe it would make sense to ask the config location before we show the "`Welcome to the Streamr Network`" message. 

Then we could maybe remove the "Broker Config Wizard ran succesfully" as the success is implied by the "`Welcome to...`" message. And "`Stored config under ${destinationPath}"` is not needed, because we show the location in the "`streamr-broker ${destinationPath}`" line below.

Reply:
Maybe worth adding a bullet point to PR description about this simplification.